docs/components/renderer.md

```diff
@@ -82,13 +82,20 @@ Controls the amount of foveation which renders fewer pixels near the edges of th
 when in stereo rendering mode on certain systems. The value should be in the range of 0 to 1, where
 0 is the minimum and 1 the maximum amount of foveation. This is currently supported by the Oculus Browser.
 
-### sortObjects
+### sortTransparentObjects
 
-Sorting is used to attempt to properly render objects that have some degree of transparency.
-Due to various limitations, proper transparency often requires some amount of careful setup.
-By default, objects are not sorted, and the order of elements in the DOM determines order of
-rendering. Re-ordering DOM elements provides one way of forcing a consistent behavior, whereas
-use of `renderer="sortObjects: true"` may cause unwanted changes as the camera moves.
+In may cases, the order in which objects is rendered doesn't matter much, but there are a few cases where sorting is important:
```

Comment:
This is documentation of general A-Frame object sorting behaviour, rather than this particular property.

Not sure if there's a better place for this in the docs, but I couldn't see an obvious one, and didn't want to restructure docs too much.

Reply:
Small typo: may -> many

The text seems a bit out of place compared to the rest as it's considerably more in-depth and technical. While I personally don't mind it too much, it's usefulness to new(er) users is probably limited as a result. Surprisingly the original text still mostly holds (as it already pertained to transparent objects). Touching that up and linking to a different page or resource describing it more in-depth (preferably with concrete pros and cons) would IMHO be ideal.

Reply:
What you suggest is the approach I wanted to take, but I couldn't find a good place in the docs to slip in a section on render order.  It's too detailed for me to want to put it into the core docs sections that work as a guide for newcomers.

Have looked again and I think the FAQ https://aframe.io/docs/1.4.0/introduction/faq.html is somewhere that could probably accommodate something.

Probably a question "What order does A-Frame render objects in?" though it could also be "Why are my transparent objects blocking other objects?".

Sound OK?

Reply:
Sounds okay to me, I would go for the "Why are my transparent objects blocking other objects?" or something along those lines. It's closer to the question people likely have when facing the issue, especially if they don't know it has to do with sorting. (And those that do know it has to do with sorting will likely find the `sortTransparentObjects` property first which links to the relevant FAQ question).

Reply:
I've left it as "what order..." - there are other ordering issues that are nothing to do with transparency (e.g. the hider material issues that got me into looking at this in the 1st place, and a bunch of the content has nothing to do with transparency).